src/GroupRow.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { memo } from 'react';
+import clsx from 'clsx';
+import { GroupRowRendererProps } from './types';
+import { SELECT_COLUMN_KEY } from './Columns';
+import GroupCell from './GroupCell';
+
+function GroupedRow<R, SR>({
+  id,
+  groupKey,
+  viewportColumns,
+  childRows,
+  rowIdx,
+  top,
+  level,
+  isExpanded,
+  selectedCellIdx,
+  isRowSelected,
+  eventBus,
+  onFocus,
+  onKeyDown,
+  'aria-setsize': ariaSetSize,
+  'aria-posinset': ariaPosInSet,
+  'aria-rowindex': ariaRowIndex,
```

Comment:
Don't need to extract all the props if we're going to spread `...props` back into the div anyway.